libraries/GCS_MAVLink/#GCS.cpp#

```diff
@@ -0,0 +1,55 @@
+#include "GCS.h"
```

Comment:
What is this file? (the name is really weird)

Reply:
emacs file indicating a file is currently being edited.  Fixed.